build/android/config.gni

```diff
@@ -25,3 +26,15 @@ brave_chrome_java_srcjar_deps = [
   "//brave/android:brave_config_java",
   "//brave/browser/android/preferences:java_enums_srcjar",
 ]
+
+brave_chrome_app_java_resources_deps = [
+  "//brave/build/android:android_brave_strings_grd",
+]
+
+brave_chrome_app_java_resources_dirs = [
+  "//brave/android/java/res",
+]
+
+if (enable_android_night_mode) {
```

Comment:
where do we set that variable?

Reply:
we don't, it's set in `ui/android/features.gni` and the same guard exists in chrome for `res_night` resources